bigquery/google/cloud/bigquery/table.py

```diff
@@ -616,37 +619,50 @@ def _set_properties(self, api_response):
             cleaned['expirationTime'] = float(cleaned['expirationTime'])
         self._properties.update(cleaned)
 
-    def _build_resource(self):
+    def _build_resource(self, fields=[]):
         """Generate a resource for ``create`` or ``update``."""
         resource = {
             'tableReference': {
                 'projectId': self._project,
                 'datasetId': self._dataset_id,
                 'tableId': self.table_id},
         }
-        if self.description is not None:
+        if (
+                self.description is not None and fields == [] or
+                'description' in fields):
             resource['description'] = self.description
 
-        if self.expires is not None:
+        if self.expires is not None and fields == [] or 'expires' in fields:
             value = _millis_from_datetime(self.expires)
             resource['expirationTime'] = value
 
-        if self.friendly_name is not None:
+        if (
+                self.friendly_name is not None and fields == []
+                or 'friendly_name' in fields):
             resource['friendlyName'] = self.friendly_name
 
-        if self.location is not None:
+        if self.location is not None and fields == [] or 'location' in fields:
             resource['location'] = self.location
 
-        if self.partitioning_type is not None:
+        if (
+                self.partitioning_type is not None and fields == []
+                or 'partitioning_type' in fields):
             resource['timePartitioning'] = self._properties['timePartitioning']
 
-        if self.view_query is not None:
+        if (
+                self.view_query is not None and fields == []
+                or 'view_query' in fields):
             view = resource['view'] = {}
             view['query'] = self.view_query
-            if self.view_use_legacy_sql is not None:
-                view['useLegacySql'] = self.view_use_legacy_sql
 
-        if self._schema:
+        if (
```

Comment:
We want standard SQL to be the default, so if self.view_use_legacy_sql is None and fields == [], we want to set it to False in the resource.

Reply:
I just asked @tswast about this and it sounds like we can fix this when we build a View class